core/dnsserver/zdirectives.go

```diff
@@ -55,4 +55,5 @@ var Directives = []string{
 	"whoami",
 	"on",
 	"sign",
+	"fanout",
```

Comment:
This needs to be added to plugin.cfg. zdirectives.go is generated from plugin.cfg.

Order is important ... This controls the order of operation of plugins (order in Corefile does not).  Probably should be right after forward.

Reply:
thanks, fixed.